todo/src/reducers/index.js

```diff
@@ -0,0 +1,26 @@
+import { ADD_TODO, COMPLETE_TODO } from '../actions';
+
+const initialState = {
+  todos: []};
+
+export const todosReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_TODO:
+      return Object.assign({}, state, {
+        todos: [...state.todos, action.payload]
+      });
+    case COMPLETE_TODO:
+      const id = action.payload;
+      const todos = state.todos.map(todo => {
+        if (todo.id === id) {
```

Comment:
We can do this since we know we're returning a todo no matter what, so we can eliminate our else condition. [Here](https://scotch.io/tutorials/5-tips-to-write-better-conditionals-in-javascript) is an awesome article on writing really good conditionals
```suggestion
        if (todo.id === id) todo.completed = !todo.completed;
        return todo;
```